Gemfile

```diff
@@ -1,7 +1,5 @@
-source ENV['GEM_SOURCE'] || "https://rubygems.org"
+source ENV['GEM_SOURCE'] || 'https://rubygems.org'
 
 gemspec
 
-if File.exists? "#{__FILE__}.local"
-  eval(File.read("#{__FILE__}.local"), binding)
-end
+eval(File.read("#{__FILE__}.local"), binding) if File.exist? "#{__FILE__}.local"
```

Comment:
Let's turn this one off.

Reply:
https://rubocop.readthedocs.io/en/latest/cops_style/#styleifunlessmodifier disabled